libraries/SuplaDevice/supla/network/esp_wifi.h

```diff
@@ -73,7 +98,27 @@ class ESPWifi : public Supla::Network {
   }
 
   bool connect(const char *server, int port) {
-    return client.connect(server, port);
+    
+	bool result = client.connect(server, port);
+	
+#ifdef ESP_SSL
+	/* server certificate validation */
+	if (result && fingerprint)	{
+	  if (client.verify(fingerprint, server)) {
+#ifdef SUPLA_COMM_DEBUG
+		Serial.println("certificate matches");
+#endif
+      } else {
+		client.stop();
+		result = false;
+#ifdef SUPLA_COMM_DEBUG
+        Serial.println("certificate doesn't match");
```

Comment:
is it possible to print both fingerprints here? It will be easier to debug connection problem